### ISSUE TITLE
MiniMessages support.

### ISSUE THREAD

**Issue by BlitzOffline** [Frcsty/FrozenJoin#29]
Possible switch to adventure/minimessages.

I am thinking at first we should use both a legacy serializer and minimessages. I know I know. not the best. But in the meantime we give people 1 version to use our new legacy to minimessages converter to convert and we can also fix possible bugs in the minimessages converter if any occur so the update after will remove complete legacy support with an easy and reliable way to convert.

The converter will work something like this: (at least for strings). We convert the current hex formats to &#aaFF00 then we use a legacy converter to convert it to components then we use the mini messages serializer to serialize the components to string. ez convertor idea.

**Comment by ghost** [comment]
Hey @BlitzOffline, do you have any updates for this? :D

**Comment by BlitzOffline** [comment]
Yeah. I thought about it and I don't think I'll end up doing it. Instead, I'm just thinking of adding actions supporting Adventure such as [adventuresound], [minimessage], etc.

**Comment by ghost** [comment]
> Yeah. I thought about it and I don't think I'll end up doing it. Instead, I'm just thinking of adding actions supporting Adventure such as [adventuresound], [minimessage], etc.

oh yeah actions would be nice 👍
